docs/source/markdown/podman-wait.1.md

```diff
@@ -10,8 +10,9 @@ podman\-wait - Wait on one or more containers to stop and print their exit codes
 
 ## DESCRIPTION
 Waits on one or more containers to stop.  The container can be referred to by its
-name or ID.  In the case of multiple containers, podman will wait on each consecutively.
-After the container stops, the container's return code is printed.
+name or ID.  In the case of multiple containers, Podman will wait on each consecutively.
+After all specified containers are stopped, the container's return codes are printed
```

Comment:
Still wrong. It should be ess then apostrophe: "containers' return codes". Sorry, I should've been clearer.

Reply:
Hey quick note, as long as you're updating this, could you also update lines 7 and 9 to add `...` (dot dot dot) after `container`? To make it clearer that multiple containers are allowed?

Reply:
I'm not a native speaker. I have never seen an apostrophe at the end of a word. So I assumed it was a typo.

Reply:
I'm not a native speaker either :-). See https://en.wikipedia.org/wiki/Apostrophe#Plural_possessive

Reply:
TIL :smiley: